scrapers/utils/secrets.py

```diff
@@ -0,0 +1,31 @@
+import boto3
+import json
+import logging
+
+
+def get_secret(secret_name, region="us-west-2"):
+    """
+    Retrieves secret value from a secret in AWS Secrets Manager.
+
+    Parameters:
+    - secret_name (str): The name of the secret.
+    - region (str): The AWS region where the secret is stored.
+
+    Returns:
+    - The value associated with the specified key.
+    """
+
+    client = boto3.client(service_name="secretsmanager", region_name=region)
+
+    try:
+        secret_response = client.get_secret_value(SecretId=secret_name)
+    except Exception as e:
+        logging.warning(f"Error retrieving secret: {e}")
+        return None
```

Comment:
What's going to happen if this is `None`? Think about this both in relation to the update in this PR but also elsewhere we're going to use this

Reply:
I changed this script to raise an exception to stop the script at this point, but please let me know if I misunderstood or if this is too duplicative (raising an exception as well as logging an error).